### PR TITLE
go 1.18 follow up

### PR DESCRIPTION
**Pull request by 0x656b694d**
## Description

Bump go version for static check.

## Checklist
- [x] Investigated and inspected CI test results
- [ ] ~Unit test and regression tests added~
- [ ] ~Evaluated and added CHANGELOG entry if required~
- [ ] ~Determined and documented upgrade steps~
- [ ] ~Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

If any of these don't apply, please comment below.

### CODE REVIEW

File: roxctl/helm/output/output_test.go [hunk 1]
```diff
@@ -108,23 +108,23 @@ func (suite *helmOutputTestSuite) TestValidate() {
 			outputDir: "test_output_dir",
 		},
 		"should not fail for valid chartName and non provided outputDir": {
-			chartName: common.ChartCentralServices,
+			chartName:    common.ChartCentralServices,
```
Comment (janisz) [inline roxctl/helm/output/output_test.go:111]: 🤔  
```
roxctl/helm/output/output_test.go:111: File is not `gofmt`-ed with `-s` (gofmt)
			chartName:    common.ChartCentralServices,
```

**Review by rukletsov — CHANGES_REQUESTED** [review]
I'd stick to making only necessary changes. It seems to be accepted by our CI: https://github.com/stackrox/stackrox/pull/3207

**Comment by janisz** [comment]
> I'd stick to making only necessary changes. It seems to be accepted by our CI

@rukletsov I think it's fine to change formatting. If we do not do this then this changes will randomly appear when someone runs gofmt on file and commit it.

**Review by rukletsov — APPROVED** [review]

**Pull request merged**